dao/src/testFixtures/kotlin/MockDatabaseModule.kt

```diff
@@ -85,7 +85,7 @@ fun KoinTest.verifyDatabaseModuleIncluded() {
  * Run [block] with a [mocked database module][mockDatabaseModule]. Afterward do cleanup by calling
  * [unmockDatabaseModule].
  */
-fun withMockDatabaseModule(block: () -> Unit) {
+inline fun withMockDatabaseModule(block: () -> Unit) {
```

Comment:
Maybe it's worth spending a sentence in the commit message on why you decided to inline these instead of making them `suspend`, too?

Reply:
Extended the commit message and also the comment for these functions.